checkov/secrets/plugins/entropy_keyword_combinator.py

```diff
@@ -220,18 +220,24 @@ def analyze_line(
                         kwargs=kwargs
                     )
                 else:
+                    # preprocess line before detecting secrets - add quotes on potential secrets to allow triggering
+                    # entropy detector
                     for pt in keyword_on_key:
                         if pt.secret_value:
                             quoted_secret = f"\"{pt.secret_value}\""
                             if line.find(quoted_secret) < 0:    # replace potential secret with quoted version
                                 line = line.replace(pt.secret_value, f"\"{pt.secret_value}\"", 1)
-                    return self.detect_secret(
+                    detected_secrets = self.detect_secret(
                         scanners=self.high_entropy_scanners_iac,
                         filename=filename,
                         line=line,
                         line_number=line_number,
                         kwargs=kwargs
                     )
+                    # postprocess detected secrets - filter out potential secrets on keyword and re-run secret detection
+                    # on their value only
+                    self.postprocess_secrets(detected_secrets, line)
```

Comment:
Let's try a more explicit name? For example:
```suggestion
                    self.remove_fp_secrets_in_keys(detected_secrets, line)
```